iconservice/icon_constant.py

```diff
@@ -229,6 +230,25 @@ class IconNetworkValueType(Enum):
     SCORE_BLACK_LIST = b'score_black_list'
     IMPORT_WHITE_LIST = b'import_white_list'
 
+    IREP = b'irep'
+
+    @classmethod
+    def gs_migration_type_list(cls) -> list:
+        return [
+            cls.SERVICE_CONFIG,
+            cls.STEP_PRICE,
+            cls.STEP_COSTS,
+            cls.MAX_STEP_LIMITS,
+            cls.REVISION_CODE,
+            cls.REVISION_NAME,
+            cls.SCORE_BLACK_LIST,
+            cls.IMPORT_WHITE_LIST,
+        ]
+
+    @classmethod
+    def gs_migration_count(cls) -> int:
+        return len(cls.gs_migration_type_list())
```

Comment:
I think Using 'INVEngine._get_gs_data_mapper' is more appropriate.

Reply:
In IconNetworkValueType, I want to define the type that needs to be migrated. This is because you need to have an INVEngine instance to access INVEngine._get_gs_data_mapper. You can access IconNetworkValueType.gs_migration_type_list from anyware.